js/main.js

```diff
@@ -291,13 +295,20 @@ function generateRecipeDOM(recipe) {
   $heartIcon.setAttribute('alt', 'favorites icon');
   $heartIcon.className = 'favorite-icon';
 
-  var $anchorWrap = document.createElement('a');
-  $anchorWrap.setAttribute('href', '#');
-  $anchorWrap.appendChild($heartIcon);
+  var $noHeartIcon = document.createElement('img');
+  $noHeartIcon.setAttribute('src', 'images/heart-no.svg');
+  $noHeartIcon.setAttribute('alt', 'unfavorite icon');
+  $noHeartIcon.className = 'unfavorite-icon transparent';
+  if (!notInFavorites(recipe)) $noHeartIcon.className = 'unfavorite-icon';
```

Comment:
This is the only place I can see you using the `notInFavorites` function, and you are immediately flipping its boolean value using `!`. This is a good indicator that the function's goal should be swapped. Instead of `notInFavorites`, you should rewrite the function to be `isInFavorites`.